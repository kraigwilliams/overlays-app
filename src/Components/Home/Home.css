.home-page-top{
  height:100%;

  /* width: 60%; */
  width:100%;
  margin-left: auto;
  margin-right: auto;
  display: flex;
  flex-direction: column;
/* justify-content: baseline; */
justify-content: center;
   align-items: center;
  flex-wrap:wrap;
padding-top: 0px;
  /* height: 90%; */
  
}
.home-page{
  margin-left:auto;
  margin-right:auto;
  flex-wrap: wrap;
  width:80%;
  display:flex;
  justify-content: center;
  align-items:center;
  color:blue;
}
.home-page *{
  font-size: 50px;
  /* font-weight: 900; */
  
}

.home-left {
  
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: flex-start;
  /* margin-right: 100px; */
  text-align: left;
}

.home-right{
    margin-top:300px;
    display:flex;
  flex-direction: column;
  text-align: center;
  align-items: center;
  justify-content: center;

} 
.home-bottom{
  margin-top:200px;
}



.underline{
  text-decoration: underline;
  text-align: center;

}
.home-left p{
  margin-top: 0px;
  text-align:left;
}


  
@media only screen and (min-width: 768px) {
  .home-page *{
    color:red;
    display: flex;
    flex-direction: row;
    
  }
}
