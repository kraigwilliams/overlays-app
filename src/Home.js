import React from 'react';
import {Link} from 'react-router-dom'

 


function Home(){
return(
    <>

    <h3>Share Your World</h3>


    <p>
     <Link to="#">Create Account</Link> or <Link to="#">Login</Link>

    </p>
    </>
)
}

export default Home;