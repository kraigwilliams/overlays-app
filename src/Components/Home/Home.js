import React from 'react';
import {Link} from 'react-router-dom';
import './Home.css';
 


function Home(){
return(
    <div className="home-page">
<div className="home-left">
<p>Save Your Opinions on any page you visit on the web.</p>
<p>Leave Your View on Everything</p>

    <p>Save What's Important to You</p>
</div>


<div className="home-right">
    


    <p>
    <Link className="options" to="/register">Create Account</Link> or <Link className="options" to="/login">Login</Link>

    </p>
    </div>

    </div>
)
}

export default Home;