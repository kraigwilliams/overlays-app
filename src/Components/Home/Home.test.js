import React from 'react';
//import { render } from '@testing-library/react';
import Home from './Home'
import {BrowserRouter} from 'react-router-dom'
import ReactDOM from  'react-dom';
import renderer from 'react-test-renderer';

describe('<Header/>',()=>{
  it ('is able to render',()=>{
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><Home/></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<BrowserRouter><Home/></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

  
})