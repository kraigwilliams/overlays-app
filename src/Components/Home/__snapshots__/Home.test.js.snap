// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders the UI as expected 1`] = `
<div
  className="home-page"
>
  <div
    className="home-page-top"
  >
    <div
      className="home-left"
    >
      <p>
        <i
          className="fas fa-bookmark"
        />
        Save Your Opinions on any Page You Visit on the Web.
      </p>
      <p>
        <i
          className="fas fa-bookmark"
        />
        Save Your Views on Everything
      </p>
      <p>
        <i
          className="fas fa-bookmark"
        />
        Save What's Important to You
      </p>
    </div>
    <div
      className="home-right"
    >
      <a
        className="options"
        href="/register"
        onClick={[Function]}
      >
        <span
          className="underline"
        >
          Create Account
        </span>
      </a>
      <span>
        or
      </span>
      <a
        className="options"
        href="/login"
        onClick={[Function]}
      >
        <span
          className="underline"
        >
          Login
        </span>
      </a>
    </div>
  </div>
  <div
    className="home-bottom"
  >
    <p>
      To demo Overlays App you may login with:
      <br />
      username: kwill
      <br />
      password: 1@Thinkful
    </p>
  </div>
</div>
`;
